.github/workflows/centraldb_angular_docker_publish.yaml

```
@@ -0,0 +1,57 @@
+name: Build & Publish dashboard-Angular Docker image
+on:
+  push:
+    branches:
+      - main
+      - v*-branch
+    paths:
+      - components/dashboard-angular/**
+      - releasing/version/VERSION
+
+env:
+  IMG: ghcr.io/kubeflow/dashboard/dashboard-angular
+
+jobs:
+  push_to_registry:
+    name: Build & Push image to GHCR
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - uses: dorny/paths-filter@v3
+      id: filter
+      with:
+        base: ${{ github.ref }}
+        filters: |
+          version:
+            - 'releasing/version/VERSION'
+
+    - name: Login to GHCR
+      uses: docker/login-action@v3
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Run dashboard-Angular build/push
+      run: |
+        cd components/dashboard-angular
```

```suggestion
        cd ${{ env.COMPONENT_PATH }}
```

see: https://github.com/kubeflow/dashboard/pull/115/files#r2288233149